pkg/alexa/request.go

```diff
@@ -141,16 +254,45 @@ type RequestType string
 
 // Request type constants.
 const (
-	// TypeLaunchRequest
+	// TypeLaunchRequest string that represents a launch request
```

Comment:
```suggestion
	// TypeLaunchRequest defines a launch request type.
```

Apply below as well.